src/extension.ts

```diff
@@ -48,7 +48,7 @@ let crashCount = 0;
 
 export async function activate(context: vscode.ExtensionContext): Promise<void> {
   const manifest = context.extension.packageJSON;
-  reporter = new TelemetryReporter(context.extension.id, manifest.version, manifest.appInsightsKey);
+  reporter = new TelemetryReporter(manifest.appInsightsKey);
```

Comment:
Where does it get the extension ID & version from now? Is that automagically read at runtime from `package.json` or something?

Reply:
With [0.7.5](https://github.com/microsoft/vscode-extension-telemetry/releases/tag/v0.7.5) they removed the need to provide these parameters. I assume the new VS Code telemetry API handles this internally.

Reply:
As long as you confirmed the VS Code telemetry API handles that and the events land in AppInsights in the expected shape (i.e. under the expected ID & version), then 👍🏻 LGTM